src/Prefix.jl

```diff
@@ -297,6 +297,9 @@ function setup(source::SetupSource{ArchiveSource}, targetdir, verbose; tar_flags
 end
 
 function setup(source::SetupSource{FileSource}, target, verbose)
+    if isdir(target)
+        target = joinpath(target, basename(source.path))
+    end
```

Comment:
The issue was that `cp()` was trying to copy a file to a directory.  This will happen anytime someone lists an archive that is not recognized as an archive in the wizard.